templates/eap-cd-amq-s2i.json

```diff
@@ -842,7 +842,7 @@
                                         "value": "${MQ_QUEUES}"
                                     },
                                     {
-                                        "name": "AMQ_TOPICS",
+                                        "name": "AMQ_ADDRESSES",
```

Comment:
I might be wrong but I think this still should be AMQ_TOPICS.

Reply:
AMQ_TOPICS doesn't exist in the example AMQ template that I used for reference.

https://github.com/jboss-container-images/jboss-amq-7-broker-openshift-image/blob/amq71-dev/templates/amq-broker-71-basic.yaml#L191

Reply:
Also, see here: https://github.com/jboss-container-images/jboss-amq-7-broker-openshift-image/blob/amq71-dev/modules/amq-launch/added/launch.sh#L30

And the example for Artemis: https://github.com/apache/activemq-artemis/blob/master/examples/features/standard/topic/src/main/resources/activemq/server0/broker.xml#L51

Reply:
Has anybody mentioned to them that they've changed the API for the image from 6.x to 7.x?

Reply:
@luck3y, AMQ Broker 7 does not distinguish between JMS and non-JMS configuration.  The thought was that the model differences between AMQ 6 and AMQ Broker 7, should be considered during migration.  https://access.redhat.com/documentation/en-us/red_hat_amq/7.1/html-single/migrating_to_red_hat_amq_7/index#message_address_changes_in_amq_7
@RoddieKieley would know if there is talk about changing it.

Reply:
@vbusch It works fine, so no complaints from me :)